src/tachimawari/joint/node/joint_manager.cpp

```diff
@@ -83,14 +97,20 @@ bool JointManager::torque_enable(bool enable)
 
 bool JointManager::torque_enable(const std::vector<Joint> & joints, bool enable)
 {
-  if (joints.size()) {
-    for (const auto & joint : joints) {
-      return control_manager->write_packet(
-        joint.get_id(), protocol_1::MX28Address::TORQUE_ENABLE, enable);
-    }
+  if (std::any_of(
+      joints.begin(), joints.end(), [&](Joint joint) {
```

Comment:
I suggest to use `this` rather than `&` because this lambda uses member of object (class), `control_manager`. Yeah, of course both are the same.

Reply:
this lambda also needs `enable`, that's why I put `&` here